feg/protos/s8_proxy.proto

```diff
@@ -118,31 +117,32 @@ message Fteid {
 }
 
 message CreateSessionResponsePgw {
-    Fteid pgw_fteid_u = 2 ;
     string subscriber_ip = 3;
```

Comment:
It might be helpful to put in comments where these values come from. And 3gpp spec.

Reply:
Hey @themarwhal the 3gpp specs reference are in the top of this file at
http://github.com/magma/magma/blob/master/feg/protos/s8_proxy.proto#L33-L33

Reply:
oooo didn't see!